backup.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# determine which environment to backup
+if [[ "${1:-}" != "testing" && "${1:-}" != "production" ]]; then
+    echo "Please provide an environment as the first argument - either \"testing\" or \"production\"!"
+    exit 1
+fi
+
+env="${1:-}"
+source .env.${env}
+
+# define place and number of backups
+mediaDirectoryInContainer="/images/uploads"
+directoryOfBackupInContainer="/backup"
+directoryOfBackup=$BACKUP_PATH
+maxNumberOfBackups=7
+
+date=$(date +%Y-%m-%d-%H-%M-%S)
+
+# create a new backup
+mkdir -m a+rw "$directoryOfBackup"/"$date"
```

Comment:
I have to increase the permissions two times because the user starting the backup process is different from the user in the docker container. Is there a better way to do this?